stable/database/templates/deployment.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     kubectl.kubernetes.io/default-logs-container: engine
 {{- include "database.loadBalancerConfig" . | indent 4 }}
 {{- include "database.automaticProtocolUpgrade" . | indent 4 }}
+    {{- if eq (include "defaulttrue" .Values.database.primaryRelease) "false" }}
+    nuodb.com/database-secondary-release: "true"
```

Comment:
Adding `nuodb.com/database-secondary-release: false` instead of `nuodb.com/database-primary-release: true` is done mainly for backward compatibility for cases where you have a newer version of NuoDB with an older version of the Helm Charts. This way any _Deployment_ that doesn't have the `nuodb.com/database-secondary-release` is considered installed from the primary Helm release.